### PR TITLE
Sequelize Team and Season Models and Migrations

### PR DESCRIPTION
**Pull request by mienaikoe**

### CODE REVIEW

File: server/sequelize/models/Person.js [hunk 1]
```diff
@@ -48,6 +48,15 @@ module.exports = (sequelize, DataTypes) => {
       foreignKey: 'person_id',
       as: 'settings'
     });
+    Person.belongsToMany(models.Season, {
+      through: 'season_people',
+      foreignKey: 'person_id',
+      as: 'seasons'
+    })
+    Person.hasMany(models.Team, {
+      foreignKey: 'owner',
```
Comment (k3ntako) [inline server/sequelize/models/Person.js]: isn't the key `owner_id`

File: server/sequelize/models/Team.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    underscored: true,
+    indexes: [
+      { unique: true, fields: [ 'email' ] },
+    ],
+    tableName: 'teams', // Sequelize defaults to pluralized version of model name (i.e., Sessions)  
+  });
+  Team.associate = function(models) {
+    Team.belongsTo(models.Person, {
+      foreignKey: 'owner',
+      as: 'owner_id',
```
Comment (k3ntako) [inline server/sequelize/models/Team.js]: did you flip these? `foreignKey` should be `'owner_id'` and `as` should be `'owner'`?

File: server/sequelize/models/Team.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      foreignKey: 'owner',
+      as: 'owner_id', 
+    })
+    Team.hasMany(models.Season, {
+      foreignKey: 'team_id',
+      as: 'seasons', 
+    })
+    Team.belongsTo(models.Season, {
+      foreignKey: 'active_season',
+      as: 'active_season_id',
```
Comment (k3ntako) [inline server/sequelize/models/Team.js]: same as above. are these flipped?

File: server/sequelize/models/Team.js (hunk 2 above)
Comment (mienaikoe) [reply server/sequelize/models/Team.js]: good catch!

**Pull request merged**